ostree-compliance-mode.spec

```diff
@@ -0,0 +1,37 @@
+Name:           ostree-compliance-mode
+Version:        0.1
+Release:        1%{?dist}
+Summary:        ostree-compliance-mode
+
+License:        GPLv2
+URL:            https://github.com/ericcurtin/%{name}
+Source0:        %{url}/archive/main.tar.gz
+
+BuildRequires: meson
+BuildRequires: gcc-c++
+BuildRequires: ninja-build
+
+%description
+%{summary}
+
+%prep
+%autosetup
+
+%build
+%meson
+%meson_build
+
+%check
+%meson_test
+
+%install
+%meson_install
+
+%files
+%license LICENSE
+%doc README.md
+%{_bindir}/ostree-compliance-mode
```

Comment:
You'll need ostree-compliance-mode-helper also, both with the correct permissions.

Might need to autogenerate /etc/ostree-compliance-mode.conf also at install time with the current rpm-ostree/ostree ref.

Reply:
This section looks great!